### DISCUSSION TITLE
Inav realese for  Flywoo HEX F745 FC  hexacopter

### DISCUSSION THREAD

**Discussion by elbartnox** [iNavFlight/inav#7529] [Q&A]
I own this Fc mounted on a Flywoo Hexplorer LR4 Hexacopter. (GOKU HEX F745).
Will a compatible version be released for this fc?
Thanks

**Comment by MrD-RC** [comment]
https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md
